pkg/util/arm/arm.go

```diff
@@ -104,3 +106,49 @@ func FixupDepends(subscriptionID, resourceGroup string, template map[string]inte
 		}
 	}
 }
+
+// FixupSDKMismatch adds missing configurations and objects into the generated template
+func FixupSDKMismatch(template map[string]interface{}) error {
+	for _, resource := range jsonpath.MustCompile("$.resources.*").Get(template) {
+		typ := jsonpath.MustCompile("$.type").MustGetString(resource)
+		name := jsonpath.MustCompile("$.name").MustGetString(resource)
+		// inject management vnet conigurations into the VNET config
+		if typ == "Microsoft.Network/virtualNetworks" && name == VnetName {
+			jsonpath.MustCompile("$.apiVersion").Set(resource, "2019-06-01")
+			for _, subnet := range jsonpath.MustCompile("$.properties.subnets.*").Get(resource) {
+				name := jsonpath.MustCompile("$.name").MustGetString(subnet)
+				if name == VnetManagementSubnetName {
+					jsonpath.MustCompile("$.properties.privateEndpointNetworkPolicies").Set(subnet, "Disabled")
+					jsonpath.MustCompile("$.properties.privateLinkServiceNetworkPolicies").Set(subnet, "Disabled")
+				}
+			}
+		}
+
+	}
+	return nil
+}
+
+// Generate generates ARM template, based on resource interface provided
+// This version of Generate will not do any version fixup
+func Generate(ctx context.Context, subscriptionID, resourceGroup string, resource []interface{}) (map[string]interface{}, error) {
```

Comment:
should this function be under pkg/fakerp ?

Reply:
I dont mind where this lands :)

Reply:
move to `pkg/fakerp/arm`